neo4j/metadata.csv

```diff
@@ -1,119 +1,399 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
-neo4j.bolt.accumulated_processing_time,gauge,,,,The accumulated time worker threads have spent processing messages.,0,neo4j,bolt accumulated processing time,
```

Comment:
This metric (and a few others) are not included anymore in your new metadata.csv. Is this correct?